src/main/java/net/revelc/code/zookeeper/maven/plugin/StartZooKeeperMojo.java

```diff
@@ -37,7 +37,7 @@
 @Mojo(name = "start", defaultPhase = LifecyclePhase.PRE_INTEGRATION_TEST, threadSafe = true)
 public class StartZooKeeperMojo extends AbstractZooKeeperMojo {
 
-  @Parameter(defaultValue = "${project.build.directory}/zmp", readonly = true)
+  @Parameter(defaultValue = "${project.build.directory}/zmp", property = "zmp.directory")
```

Comment:
Cool. In case you didn't know, you can push to the same branch you made the original pull request with, and it will automatically update the pull request with the changes. It works with new commits and with force pushes if you replace the old commits.